### ISSUE TITLE
Possible Performance Improvement

### ISSUE THREAD

**Issue by pramitchoudhary** [datascienceinc/lime#3]
Based on the benchmarks done here
http://stackoverflow.com/questions/20277982/fastest-pairwise-distance-metric-in-python. 
Using  _scipy.spatial.distance.pdist_ vs _sklearn.metrics.pairwise_distances_ could help in improving the computation for computing _euclidean distance_ distance. Need to test it out.
